api/documentation/controller.py

```diff
@@ -454,9 +454,35 @@ def addSchemes(self):
             }
         )
         self.addComponent(
-            'schema', 'BulkCirculation', 'list',
+            'schema', 'BulkCirculationEvents', 'array',
             {'items': 'string'}
```

Comment:
Should be `type: string` here. The documentation actually works, just throws an error in the UI!

Reply:
fixed